src/parse-font-family-name-value.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TokenType, tokenize } from '@csstools/css-tokenizer'
+
+const IGNORED_TOKEN_TYPES: ReadonlySet<TokenType> = new Set([
+  TokenType.Comment,
+  TokenType.Whitespace,
+  TokenType.EOF,
+])
+
+export const parseFontFamilyNameValue = (string: string): string | null => {
+  const tokens = tokenize({ css: string }).filter(
+    ([tokenType]) => !IGNORED_TOKEN_TYPES.has(tokenType),
+  )
+
+  if (tokens.length === 0) {
+    return null
+  }
+
+  if (tokens.length === 1 && tokens[0] && tokens[0][0] === TokenType.String) {
```

Comment:
Last question I think! Is it actually possible for the `tokens[0]` part to return false? From what I can tell it should always be non-falsy, but maybe I'm missing something

https://github.com/csstools/postcss-plugins/blob/main/packages/css-tokenizer/src/tokenizer.ts#L24
https://github.com/csstools/postcss-plugins/blob/main/packages/css-tokenizer/src/interfaces/token.ts

Reply:
It can never be false, but TypeScript fails to correlate a length of 1 with there is a value at index 0.

Is it ok to do `tokens[0]![0]`?

I am always a bit hesitant to drop a `!` into someone else's codebase :)

Reply:
Ah right, yeah I have to strict null checks on to keep me on my toes, but I'm fine adding `!` if something is provably not null. Good to add, thanks for checking!

Reply:
You're welcome!
Done